external/anomaly/ote_anomalib/exportable_code/anomaly_classification.py

```diff
@@ -51,9 +54,15 @@ def postprocess(self, outputs: Dict[str, np.ndarray], meta: Dict[str, Any]) -> n
             np.ndarray: Resulting image resized to original input image size
         """
         outputs = outputs[self.output_blob_name].squeeze()
+        anomaly_map = outputs[self.output_blob_name].squeeze()
+        pred_score = anomaly_map.reshape(-1).max()
```

Comment:
Where is pred_score used?

Reply:
ah, yes. Forgot to add a code block

Reply:
I added some processing steps, but still the pred_score is unused. I'm not sure how the exportable code is supposed to access the predicted scores. The design is a bit confusing.

Reply:
Actually good point. Looks like I added this in the cdf normalization exportable code but it is not needed there. I guess that's the source of confusion here. We can safely remove pred_score from both as we don't append it to the metadata.

Reply:
But how does the exportable code get the normalized image-level scores then?

Reply:
Now that I look at it, the exportable code is wrong. We need to add it to the metadata and modify `prediction_to_annotation_converter`. Otherwise it uses max(predictions) which will always be 100% for anomalous images. I can create a PR for this branch and the cdf one tomorrow as testing might take some time.